system/jlib/jstats.cpp

```diff
@@ -933,9 +933,26 @@ static const constexpr StatisticMeta statsMetaData[StMax] = {
     { CYCLESTAT(LeafFetch) },
     { TIMESTAT(BlobFetch) },
     { CYCLESTAT(BlobFetch) },
+    { SIZESTAT(PeakSpillFile) },
+    { TIMESTAT(AgentQueue) },
+    { CYCLESTAT(AgentQueue) },
```

Comment:
Did we miss a few before ?

Reply:
This is to make the upmerge work and ensure the new stats values have the same value moving forward